http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/leoPubsubMessageSubscriberModels.scala

```diff
@@ -120,6 +127,25 @@ object LeoPubsubMessage {
       )
   }
 
+  final case class CreateAppMessage(cluster: Option[CreateCluster],
+                                    appId: AppId,
+                                    nodepoolId: NodepoolLeoId,
+                                    project: GoogleProject,
+                                    createDisk: Boolean,
+                                    traceId: Option[TraceId])
+      extends LeoPubsubMessage {
+    val messageType: LeoPubsubMessageType = LeoPubsubMessageType.CreateApp
+  }
+
+  final case class DeleteAppMessage(appId: AppId,
+                                    nodepoolId: NodepoolLeoId,
+                                    project: GoogleProject,
+                                    deleteDisk: Boolean,
+                                    traceId: Option[TraceId])
```

Comment:
`traceId` probably doesn't need to be optional

Reply:
that param is inherited from the trait. Not sure why its optional, I think some LeoPubsubMessages may be a bit older and not pass along a traceId. I'll look at it in depth in my next PR using the messages, and see if I can make it non-optional in the parent trait